source/github/access.go

```diff
@@ -59,7 +59,7 @@ func (c *client) RepoAccess(u *library.User, org, repo string) (string, error) {
 // TeamAccess captures the user's access level for a team.
 func (c *client) TeamAccess(u *library.User, org, team string) (string, error) {
 	logrus.Tracef("Capturing %s access level to team %s/%s", u.GetName(), org, team)
-
+	team = strings.ToLower(team)
```

Comment:
Do we need this `strings.ToLower()`?

Reply:
Needed that for the database query, I moved the lowercase down to where the query is run and removed it from this function.